app/cdap/components/Administration/TetheringTabContent/TetheringTable/utils.tsx

```diff
@@ -45,22 +45,45 @@ export const getIconForStatus = (status: string, isForPendingReqs: boolean) => {
 };
 
 export const getTransformedTableData = (tableData: IConnection[]) => {
-  return tableData.map((conn) => ({
-    requestTime: dateTimeFormat(conn.requestTime, {
-      year: 'numeric',
-      month: 'numeric',
-      day: 'numeric',
-      hour: 'numeric',
-      minute: 'numeric',
-    }),
-    description: conn.metadata?.description,
-    gcloudProject: conn.metadata?.metadata?.project,
-    instanceName: conn.name,
-    region: conn.metadata?.metadata?.location,
-    allocationData: conn.metadata?.namespaceAllocations,
-    status: conn.connectionStatus,
-    highlighted: false,
-  }));
+  return tableData.map((conn) => {
+    // Avoid null pointer exception
```

Comment:
Jest did not like optional chaining (I think it's due to our node version as Sean mentioned once).

Reply:
yeah we can't do optional chaining for the time being